src/command-configs/fetchCommandsConfiguration.ts

```diff
@@ -0,0 +1,62 @@
+import { Mutex } from "async-mutex"
+import fs from "fs"
+import path from "path"
+
+import { cloneCommandBotScripts } from "src/command-configs/cloneCommandBotScripts"
+import { collectCommandConfigs } from "src/command-configs/collectCommandConfigs"
+import { getScriptsRepoRevision } from "src/command-configs/getScriptsRepoRevision"
+import { renderHelpPage } from "src/command-configs/renderHelpPage"
+import { CommandConfigs, FetchCommandConfigsResult } from "src/command-configs/types"
+import { config } from "src/config"
+import { LoggerContext } from "src/logger"
+import { DOCS_DIR, DOCS_PATH } from "src/setup"
+import { CommandRunner } from "src/shell"
+
+export const PIPELINE_SCRIPTS_REF = "PIPELINE_SCRIPTS_REF"
+
+export async function fetchCommandsConfiguration(
```

Comment:
moved & simplified from https://github.com/paritytech/command-bot/blob/79223d9d98ae9c5be92f2affdb985deeb4666c8a/src/commands.ts 
\+ added renderHelpPage step (line: 45)
\+ returning the commitHash of cloned configs, so could be used to form a correct URL to docs